rotkehlchen/db/upgrades/v50_v51.py

```diff
@@ -46,4 +46,28 @@ def _rename_event_identifier_to_group_identifier(write_cursor: 'DBCursor') -> No
         )
         write_cursor.switch_foreign_keys('ON')
 
+    @progress_step(description='Create Lido CSM support tables.')
+    def _create_lido_csm_tables(write_cursor: 'DBCursor') -> None:
+        """Create Lido CSM specific tables and indexes."""
```

Comment:
```suggestion
        """Create Lido CSM specific tables."""
```
this is just creating tables.